CHANGELOG.md

```diff
@@ -14,6 +14,17 @@ and includes an additional section for migration notes.
 - *Security* - Vulnerabilities fixes and changes.
 
 ## [Unreleased]
+### Changed
+- *ORCA-537*
+  - Renamed step-function `OrcaCopyToGlacierWorkflow` to `OrcaCopyToArchiveWorkflow`.
+  - Renamed lambda `copy_to_glacier` to `copy_to_archive`, or `PREFIX_copy_to_orca` if looking at the name in AWS.
```

Comment:
Maybe reword this to **Renamed the AWS lambda `PREFIX_copy_to_glacier` to `PREFIX_copy_to_orca`. In the ORCA repository the internal task name has been changed from `copy_to_glacier` to `copy_to_archive`**

Reply:
Changed